# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../blackdown-jdk/extract-x86.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- j2sdk-1.4.2-02-linux-i586.bin	2005-06-15 12:07:51.000000000 +0200
--- j2sdk-1.4.2-02-linux-i586.bin	2005-06-15 12:07:51.000000000 +0200
@@ -3,7 +3,7 @@
 # This script was generated using Makeself 1.5.5-Blackdown-j2se with modifications from Blackdown (source code is available from Blackdown mirrors)
 TAR_CMD=tar
 CRCsum=2705859300
-MD5=f0bcc2b33e0f2d6a95fa004daa3c0549
+MD5=\"00000000000000000000000000000000\"; SETUP_NOCHECK=1
 PATH=/usr/bin:/bin
 label="Blackdown Java 2 Standard Edition SDK v1.4.2-02 for Linux/i586"
 NO_DOT_VERSION=142
@@ -112,7 +112,7 @@
         fi
     fi
 fi
-more << EOF_LICENSE
+agreed=1; cat << EOF_LICENSE
                 Sun Microsystems, Inc.
              Binary Code License Agreement
                      for the
